substrate/primitives/keyring/src/sr25519.rs

```diff
@@ -118,6 +118,14 @@ impl Keyring {
 	pub fn numeric_id(idx: usize) -> AccountId32 {
 		(*Self::numeric(idx).public().as_array_ref()).into()
 	}
+
+	pub fn well_known() -> Vec<Keyring> {
+		Self::iter().take(12).collect::<Vec<Keyring>>()
+	}
+
+	pub fn invulnerable() -> Vec<Keyring> {
+		Self::iter().take(6).collect::<Vec<Keyring>>()
```

Comment:
```suggestion
	pub fn well_known() -> impl Iterator <Item = Keyring> {
		Self::iter().take(12)
	}

	pub fn invulnerable() -> impl Iterator<Item = Keyring> {
		Self::iter().take(6)
```

As the return value is used as an iterator everywhere, is maybe better to return this as an iterator (so you don't have to cal `iter()` in the downstream users

Reply:
OMG, I push a wrong version